README.md

```diff
@@ -50,12 +49,6 @@ See [Release procedure](CONTRIBUTING.md#release-procedure) for more information
 ### Public
 
 - [umee-1](networks/umee-1)
-- [umeemania-1](networks/umeemania-1)
```

Comment:
Unrelated: can add `canon-2` in place of `umeemania-1`

Reply:
done at https://github.com/umee-network/umee/pull/1557/commits/f18ed6dde7502bb9ee0af69eb0efd691d53c8d8f